### PR TITLE
[PROD-623] Hid dataCatalog information behind the feature flag

### PR DESCRIPTION
**Pull request by lu-c**
I think this should fix the problem on prod, but I'm unable to completely reproduce locally.

### CODE REVIEW

File: src/pages/library/dataBrowser-utils.js [hunk 1]
```diff
@@ -71,6 +71,7 @@ const extractTags = snapshot => {
 }
 
 export const useDataCatalog = () => {
+  const skip = !isDataBrowserVisible()
```
Comment (dmohs) [inline src/pages/library/dataBrowser-utils.js]: Var name suggestion?
```suggestion
  const skipMetadata = !isDataBrowserVisible()
```

**Review by dmohs — APPROVED** [review]
One naming suggestion, but not a big deal if getting this merged is urgent.

**Review by zarsky-broad — APPROVED** [review]
Could you inline `skip`?

**Comment by lu-c** [comment]
Nice thank you everyone!

**Pull request merged**